src/components/form/templates/field.ts

```diff
@@ -1,19 +1,47 @@
-import React from 'react';
-import { FormLayoutOptions } from '../form.types';
+import React, { CSSProperties } from 'react';
+import { GridLayoutOptions } from '../form.types';
+import { isObjectType } from '../schema';
 
 export const FieldTemplate = (props) => {
     let classNames = props.classNames;
-    const layout: FormLayoutOptions = props.schema.lime?.layout;
+    const colSpan = getColSpan(props.schema);
+    const rowSpan = getRowSpan(props.schema);
+    let style: CSSProperties;
 
-    if (layout?.span) {
-        classNames += ` limel-form-layout-span--${layout.span}`;
+    if (colSpan) {
+        classNames += ` limel-form-layout-colspan--${colSpan}`;
+    }
+
+    if (rowSpan) {
+        style = {
+            gridRow: `span ${rowSpan}`,
+            height: `calc(var(--min-height-of-one-row) * ${rowSpan})`,
```

Comment:
Min height wouldn't help. We initially started with that. But we came to conclusion that we should write `height` for text area to work for example. I also tested having both `max-height` and `height`. But no difference that I could see. Seems `height` is enough for now. Or what do you think?

Reply:
This is good enough for now, but I'm leaving this unresolved, as I might want to take a quick look at it later on. (And it would be near impossible to find among all other resolved comments 😅)

Reply:
⚡